ProjectLighthouse/Pages/Layouts/BaseLayout.cshtml

```diff
@@ -19,6 +19,7 @@
 
         @if (Model.User.IsAdmin)
         {
+            Model.NavigationItems.Add(new PageNavigationItem("Reports", "/reports/0", "exclamation circle"));
```

Comment:
Hmm, I would have this somewhere in the admin panel.

Reply:
Moved to the admin panel now, my original idea was that reports could be viewed by moderators so I put them in a more convenient place.

Reply:
Ah, makes sense. Yeah, once a moderation role is implemented there should be a dedicated mod panel.